utils/utils.go

```diff
@@ -41,4 +43,66 @@ func GetAbsPath(path string) string{
 		}
 	}
 	return result
-}
\ No newline at end of file
+}
+
+func GetCurrentServerVersion() (string, string) {
+	var version, product string
+	rhndefault := "/etc/rhn/rhn.conf"
+	webpath := "/usr/share/rhn/config-defaults/rhn_web.conf"
+	altpath := "/usr/share/rhn/config-defaults/rhn.conf"
+
+	files := []string{rhndefault, webpath, altpath}
+	property := []string{"product_name", "web.product_name"}
+	product = "SUSE Manager"
+	for _, path:= range files {
+		for _, search := range property {
+			p, err := ScannerFunc(path, search)
+			if err == nil {
+				product = p
+				break
+			} else if err != nil {
+				continue
+			}
+		}
+	}
+
+	if product != "SUSE Manager" {
```

Comment:
both if code is really similar, changing just the property to get. 
Should an auxiliary method be implemented that receives a list of files to look at, and a list of properties, and returns the property or an error?